### ISSUE TITLE
Port trend chart to PF4 react

### ISSUE THREAD

**Issue by dlabrecq** [project-koku/koku-ui#223]
## Story
Port the custom trend (area) chart to PF4 react.

## Impacts
UI

## Assumptions
- Work with Michael Coker (PatternFly HTML/CSS dev) to create a PF4 theme for Victory charts

## UI Details
Currently, we have a trend chart based on Victory components. We would like to contribute this to PF4 react-core in order to have a PatternFly look and feel.

Typically, PatternFly HTML/CSS is created first. However, Michael Coker needs a component in order to create a Victory theme. This theme and chart component will be used as a basis for future chart components to be added to PF4 react.

## Acceptance Criteria

A PR has been created to contribute the area chart to PF4 react

**Comment by dlabrecq** [comment]
I created a PF4 PR to introduce various chart components for a new package, named react-charts. There are ~13 components total, used to create Victory charts such as; area, bar, donut, line, pie, & stack. (Victory doesn't have a donut, but I was able to style one to match.) The other components are required to support custom containers, legend, tooltip, etc.

Note there were two charts I did not implement. The bullet chart and heat map. Those charts will be added via a separate PR.

See PR: https://github.com/patternfly/patternfly-react/pull/868
